umbrella/Cargo.toml

```diff
@@ -538,19 +539,8 @@ with-tracing = [
 runtime = [
-	"frame-benchmarking",
-	"frame-benchmarking-pallet-pov",
-	"frame-election-provider-solution-type",
-	"frame-election-provider-support",
-	"frame-executive",
-	"frame-metadata-hash-extension",
 	"frame-support",
-	"frame-support-procedural",
-	"frame-support-procedural-tools-derive",
 	"frame-system",
-	"frame-system-benchmarking",
-	"frame-system-rpc-runtime-api",
-	"frame-try-runtime",
```

Comment:
@kianenigma @ggwpe a few tweaks:  these crates don't need to be in the runtime features as well, as they are already included in `polkadot-sdk-frame`, they were also causing some annyoning warning when compiling

Reply:
features names are not global. If the `frame-benchmarking` feature is not activated in this Cargo.toml then even if `polkadot-sdk-frame` has a feature with the same name, then the feature won't be activated. This means when compiled with feature `runtime`, the umbrella will not reexport `frame-benchmarking`. `frame-benchmarking` will only be available from `polkadot-sdk-frame`. Is it what we want?

Reply:
> frame-benchmarking. frame-benchmarking will only be available from polkadot-sdk-frame. Is it what we want?

Yeah I think that is fine to access all frames thing through polkadot-sdk-frame